### DISCUSSION TITLE
Progress on sending tabs without opening Fenix

### DISCUSSION THREAD

**Discussion by uhdotie** [mozilla-mobile/fenix#26981] [Ideas and Feature Requests]
The [Issue](https://github.com/mozilla-mobile/fenix/issues/1523) has been "locked as off-topic", which I frankly don't understand, and the [issue](https://github.com/mozilla-mobile/fenix/issues/4275) it's being tracked in has been closed. Other people have tried to open Issues about it, but all are closed with references to the locked issue, which they can't comment in or even react to.

So, is it going to be reintroduced or not? If it is, please re-open the issue so people can comment and/or react. If not, please say so. It was an infuriating removal, second only to the removal of extensions, only made worse by the positively juvenile way of dealing with requests for reintroductions. Pee or get off the pot ffs.

**Comment by kbrosnan** [comment]
[Issues/bugs are not discussion forums](https://bugzilla.mozilla.org/page.cgi?id=etiquette.html). That issue became a discussion forum. If you want to suggest features we have https://connect.mozilla.org. The issue is not closed so it is still valid thus any issues reporting the same thing are duplicated to it.

I do like that feature but we don't have it in any near term plans.

**Comment by uhdotie** [comment]
It's effectively closed, no-one can comment or even react to it, so it just sits there doing nothing, for a year now. If you don't have any plans for it, you should say so and close it; again, piss or get off the pot.

Why you removed it in the first place is beyond me, it was as ill-thought-out as the "curated" extensions. Here's an idea: consult with your userbase **before** making decisions on major feature removals. Not after.
